test/integration/scheduler/framework_test.go

```diff
@@ -578,30 +585,52 @@ func TestPostFilterPlugin(t *testing.T) {
 	numNodes := 1
 	tests := []struct {
 		name                      string
+		numNodes                  int
 		rejectFilter              bool
+		failScore                 bool
 		rejectPostFilter          bool
 		expectFilterNumCalled     int
+		expectScoreNumCalled      int32
 		expectPostFilterNumCalled int
 	}{
 		{
-			name:                      "Filter passed",
+			name:                      "Filter passed and Score success",
+			numNodes:                  3,
```

Comment:
ref: https://github.com/kubernetes/kubernetes/pull/93534#discussion_r464686653
I introduced `numNodes` in test case. set it to `3` here and others to `1`